guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,80 @@
+[id="overview-of-recommended-disaster-recovery-plans"]
+= Overview of recommended disaster recovery plans
+
+Choose a disaster recovery plan that best helps ensure the continuity of {Project} services in your deployment.
+
+Virtualizating your {ProjectServer}::
+How do I back up?:::
+Virtualize your {ProjectServer} and use the hypervisor tools to take virtual machine snapshots of the server.
+This method is suitable if you can run {Project} on top of virtualization.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, restore a virtual machine snapshot.
++
+[IMPORTANT]
+====
+During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+The IP address can change.
+====
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you created snapshosts.
+
+Active and passive {ProjectServer}, with external storage::
+How do I back up?:::
+Store the following critical data on network attached storage: content in `/var/lib/pulp` and database in `/var/lib/pgsql`.
+Replicate this storage into a different data center.
+Attach the storage to a {ProjectServer} that is a clone of the primary {ProjectServer} but runs passively.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, switch DNS records of the active {ProjectServer} with the passive {ProjectServer}.
+This ensures that the passive server becomes the active server.
+All hosts remain connected without configuration updates.
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you synchronized the data storage.
```

Comment:
Isn't the point of network attached storage that it's automatically in sync?

Reply:
Maybe we need to split this into two scenarios?

1. Network attached storage that is mounted directly by both the active and the passive
2. Network attached storage that is replicated on some interval to another location

Reply:
Is 2 "I run `rsync` via `cron` twice a day"?

Reply:
I added an internal note about the two options for attaching the storage but it looks like material for a subsequent PR that adds actual procedures.